addons/packages/fluxcd-source-controller/0.21.2/package.yaml

```
@@ -0,0 +1,27 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  name: fluxcd.source.controller.community.tanzu.vmware.com.0.21.2
```

```suggestion
  name: fluxcd-source-controller.community.tanzu.vmware.com.0.21.2
```

There are numerous places where this name would have to change.

---

For sure, let me change it!

---

I just changed the name of the package!